src/module/activity/component/Form/index.tsx

```diff
@@ -20,7 +25,7 @@ export interface SharedProps {
   fetchData: () => Promise<void>;
   selectedJuz: Option | undefined;
   setSelectedJuz: Dispatch<SetStateAction<Option | undefined>>;
-  selectedSurah: Option[] | undefined;
+  selectedSurah: Option | Option[] | undefined;
```

Comment:
cant we make it consistent as arrays?
when we have only one, just do this `[value]`

Reply:
yes that will be good, but this changes because of the behavior of this dropdown attributes, when `isMulti` is true it will return Option[] but when false it return Option

```ts
  isMulti={activity ? false : true}
```

https://github.com/RR-Ardhika/murojaah/pull/48/files#diff-48d6c570bbbb11a5d7c71b5221972361114d34ec9a09ad68b2c0c6224efb6194R71